File: C/Insertion_Sort.c
#include<stdio.h>
void main()
{
    int A[]={21,23,20,14,8};
    int a=5;
    for(int i=1;i<=a;i++)
    {
        for(int j=i;j>0;j--)
        {
            if(A[j]<A[j-1])
            {
                int temp=A[j];
                A[j]=A[j-1];
                A[j-1]=temp;
            }
        }
    }
    for(int i=0;i<a;i++)
    {
        printf("%d\t",A[i]);
    }
}

File: Javascript/insertionSort.js
/**
 * Implementation of insertion sort
 *
 * @param array
 * @returns array
 */
const insertionSort = (array) => {
  const arrayLength = array.length

  for (let i = 1; i < arrayLength; i++) {
    let j = i - 1
    let temp = array[i]

    while (j >= 0 && array[j] > temp) {
      array[j + 1] = array[j]
      j--
    }

    array[j + 1] = temp
  }
  return array
}
